customTests/azure_ib_write_bw.nhc

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+IB_WRITE_BW_EXE_PATH=/opt/azurehpc/test/nhc/perftest-4.5/ib_write_bw
```

Comment:
Why do we need to use this ib_write_bw? Is there a problem with the one installed at /usr/bin/?

Reply:
So, the reasoning is that I am not sure it is built with CUDA or not.  I build from source wrt to whether or not the VM has a GPU or not. I think Jithin mentioned that we should work the correct build method onto our images. Also, we want to be able to run Health checks on non-hpc/AI images as well. They may not have that tool installed.